src/models/pie.js

```diff
@@ -341,7 +348,11 @@ nv.models.pie = function() {
         // depreciated after 1.7.1
         donutLabelsOutside: {get: function(){return labelsOutside;}, set: function(_){
             labelsOutside=_;
-            nv.deprecated('pieLabelsOutside', 'use labelsOutside instead');
+            nv.deprecated('donutLabelsOutside', 'use labelsOutside instead');
+        }},
+        labelFormat: {get: function(){ return valueFormat;}, set: function(_) {
```

Comment:
Can you add a comment like the others so we know what the version was before it was deprecated?  Otherwise it's easy to lose track and never remove the things...